core/jobs/types/model_property.py

```diff
@@ -62,12 +76,14 @@ def __init__(self, model_class, property_obj):
             raise ValueError(
                 '%r is not a property of %s' % (property_obj, self._model_kind))
         else:
-            property_name = property_obj._name # pylint: disable=protected-access
+            # Here, ignore [attr-defined] is added because, mypy is not
+            # able to recognize '_name' attribute of ndb's models.
+            property_name = property_obj._name  # type: ignore[attr-defined]  # pylint: disable=protected-access
```

Comment:
Can we avoid this ignore? maybe by using `getattr`?

Reply:
Thanks, I removed it. By adding attribute's annotation in `ndb`'s stubs ( by following ndb's code https://github.com/googleapis/python-ndb/blob/141065067a65e5827074338c6a5f66788f563178/google/cloud/ndb/model.py#L1004).